phase1/vsphere/README.md

```diff
@@ -48,16 +48,130 @@ cd kubernetes-anywhere
 ### Enter the `kubernetes-anywhere` deployment environment:
 
 ```shell
-make docker-dev
+make docker-dev # Building docker image for first time can take few minutes.
 ```
 
 ### Start the deployment wizard:
 
 ```shell
 make deploy
 ```
+
 and fill complete the config wizard to deploy a kubernetes-anywhere cluster.
 
+### Sample config
+
+* Select the number of nodes. Master + Number of nodes will be deployed.
+```
+number of nodes (phase1.num_nodes) [4] (NEW) 8
+```
+
+* Set the cluster name. A folder with the cluster name will be created to place all the VMs.
```

Comment:
Are you sure that it creates folder?

"A folder with the cluster name will be created to place all the VMs." => "All kube VMs will be placed in Folder with cluster name"?